bundles/AdminBundle/Resources/public/js/pimcore/object/classes/data/numeric.js

```diff
@@ -111,7 +111,22 @@ pimcore.object.classes.data.numeric = Class.create(pimcore.object.classes.data.d
                 fieldLabel: t("decimal_precision"),
                 name: "decimalPrecision",
                 maxValue: 30,
-                value: datax.decimalPrecision
+                value: datax.decimalPrecision,
+                listeners: {
+                    change: function (el) {
+                        var minValueComponent = this.specificPanel.getComponent('minValue');
+                        var maxValueComponent = this.specificPanel.getComponent('maxValue');
+                        var value = parseInt(el.getValue());
+
+                        if (minValueComponent) {
+                            minValueComponent.decimalPrecision = value;
+                        }
+
+                        if (maxValueComponent) {
+                            maxValueComponent.decimalPrecision = value;
+                        }
+                    }.bind(this)
+                }
```

Comment:
```suggestion
                listeners: {
                    change: function (el, value) {
                        let minValueComponent = this.specificPanel.getComponent('minValue');
                        let maxValueComponent = this.specificPanel.getComponent('maxValue');

                        if (minValueComponent) {
                            minValueComponent.decimalPrecision = value;
                        }

                        if (maxValueComponent) {
                            maxValueComponent.decimalPrecision = value;
                        }
                    }.bind(this)
                }
```

You can pass the value into the listener directly. Can you please adapt that?

Reply:
@lukmzig Done it

Reply:
Thank you